pom.xml

```diff
@@ -20,10 +20,72 @@
     </license>
   </licenses>
 
+  <profiles>
+    <profile>
+      <id>internal-release</id>
+      <activation>
+        <property>
+          <name>useInternalRepo</name>
+          <value>true</value>
+        </property>
+      </activation>
+      <properties>
+      </properties>
+      <distributionManagement>
+        <repository>
+          <id>clx-releases</id>
+          <url>${env.REPO_URL}/repository/clx-releases-hosted/</url>
+        </repository>
+        <snapshotRepository>
+          <id>clx-snapshots</id>
+          <url>${env.REPO_URL}/repository/clx-snapshots-hosted/</url>
+        </snapshotRepository>
+      </distributionManagement>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.sonatype.plugins</groupId>
+            <artifactId>nexus-staging-maven-plugin</artifactId>
+            <version>1.6.13</version>
+            <extensions>true</extensions>
+            <configuration>
+              <serverId>clx-snapshots</serverId>
+              <nexusUrl>${env.REPO_URL}</nexusUrl>
+              <skipStaging>true</skipStaging>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
+    <profile>
+      <id>prod-repository</id>
+      <activation>
+        <property>
+          <name>!useInternalRepo</name>
+        </property>
+      </activation>
+      <properties>
+      </properties>
+      <distributionManagement>
+        <repository>
+          <id>ossrh</id>
+          <name>Maven Central Staging Repository</name>
+          <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+        </repository>
+        <snapshotRepository>
+          <id>ossrh</id>
+          <name>Maven Central Snapshot Repository</name>
+          <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+        </snapshotRepository>
+      </distributionManagement>
+    </profile>
+  </profiles>
+
   <scm>
     <url>https://github.com/sinch/sinch-java-sms</url>
-    <connection>scm:git:git://github.com/sinch/sinch-java-sms.git</connection>
-    <developerConnection>scm:git:git@github.com:sinch/sinch-java-sms.git</developerConnection>
+    <connection>scm:git:https://github.com/sinch/sinch-java-sms.git</connection>
```

Comment:
why do you change that?

Reply:
Laziness, I had a PAT setup but no SSH key. I'll change it back